mxcube3/routes/qutils.py

```diff
@@ -43,10 +44,14 @@ def node_index(node):
     elif node.get_parent():
         sample_model = node.get_parent().get_parent()
         sample = sample_model.loc_str
-        task_groups = sample_model.get_children();
+        task_groups = sample_model.get_children()
         group_list = [group.get_children() for group in task_groups]
         tlist = [task for task_list in group_list for task in task_list]
-        index = tlist.index(node)
+        # there is probably a better fix...
```

Comment:
I see, you are right its not so easy, but you agree that its strange to ask for the index of the node that is not in the queue, and we should ask our self's why that's happening. Because the problem might also have other side effects :)